CONTRIBUTORS.md

```diff
@@ -0,0 +1,77 @@
+<html>

```

Comment:
@mbad0la 
I think instead of having an adhoc script to get username and add it to Contributors.md, we should have it as an option in ```manage.py``` itself and have a JSON or [pickled file](https://docs.python.org/2/library/pickle.html) from which README must be rebuilt 

Reference : https://docs.djangoproject.com/en/1.11/howto/custom-management-commands/